examples/resources/awscc_sqs_queue/sqs_queue_dead_letter.tf

```diff
@@ -0,0 +1,12 @@
+resource "awscc_sqs_queue" "terraform_awscc_queue_deadletter" {
+  queue_name = "terraform-awscc-queue-deadletter-example"
+  redrive_allow_policy = jsonencode({
+    redrivePermission = "byQueue",
+    sourceQueueArns   = [awscc_sqs_queue.terraform_awscc_queue.arn]
```

Comment:
let's add the actual `awscc_sqs_queue` in the same `sqs_queue_dead_letter.tf` file to complete the example

Reply:
added `awscc_sqs_queue`.